topo/node/ceos/ceos.go

```diff
@@ -171,18 +163,14 @@ func (n *Node) GenerateSelfSigned(ctx context.Context, ni node.Interface) error
 		),
 	}
 
-	r, err := n.cliConn.SendConfigs(cfgs)
+	resp, err := n.cliConn.SendConfigs(cfgs)
 	if err != nil {
 		return err
 	}
 
-	if r.Failed() {
-		return ErrCliOperationFailed
-	}
-
 	log.Infof("%s - finshed cert generation", n.pb.Name)
```

Comment:
Do you want this log to happen even if resp.Failed? Previously this message would only log if the resp was not marked failed

Reply:
ah yes -- I dont think we should log if it failed, will push an update shortly!